Cargo.toml

```diff
@@ -98,7 +98,7 @@ rustls-tls-native-roots = [
 
 [dependencies]
 clickhouse-derive = { version = "0.2.0", path = "derive" }
-
+clickhouse-rowbinary = { version = "*", path = "rowbinary" }
```

Comment:
Perhaps it is more of `clickhouse-data-types-utils`, cause this logic is shared across RBWNAT and Native, and the produced AST can be used in libraries such as ch2rs.